docs/user-guide/vwii/installing-homebrewchannel.md

```diff
@@ -2,9 +2,8 @@
 
 ## Installing the Homebrew Channel
 
-1. Boot into [Tiramisu](browser-exploit).
-1. Launch the Homebrew Launcher, by opening the Mii Maker.
-1. Launch the vWii Compat Installer.
+1. Boot into [Aroma](browser-exploit).
+1. Launch the `vWii Compat Installer.rpx` on the Wii U Menu.
```

Comment:
```suggestion
1. Launch the `vWii Compat Installer` from the Wii U Menu.
```
The `.wuhb` will take priority over the `.rpx`.

Reply:
Don't know how I forgot about the WUHB. But indeed